Utility/Utility.Public/Include/Utility/Utility.hpp

```diff
@@ -48,19 +48,41 @@ namespace Utility
         SUCCESS,
         FAILED,
     };
-
+  
     /**  @brief Enum for tracking channel leave status.
      *   @details Utility::ChannelLeaveCodes leaveChannel(const std::string channelName) /
      *    return one of this codes.
      */
     enum class ChannelLeaveCodes : std::uint8_t
     {
         CHANNEL_NOT_FOUND,
-        CHANNEL_ALREADY_LEAVED,
         SUCCESS,
         FAILED,
     };
-
+  
+    /**  @brief Enum for tracking channel leave status.
+     *   @details Utility::ChannelDeleteCode deleteChannel(const std::string channelName) /
+     *    return one of this codes.
+     */
+    enum class ChannelDeleteCode : std::uint8_t
+    {
+        CHANNEL_NOT_FOUND,
+        CHANNEL_NON_USER,
```

Comment:
What does 'CHANNEL_NON_USER' mean?

Reply:
The channel can only be deleted by the creator. "the channel is not a user". I made a mistake in shortening

Reply:
it's better to be "channel owner error"

Reply:
Fix